Sources/Core/ARM/HardwareBreakpointManager.cpp

```diff
@@ -19,17 +19,17 @@ namespace ds2 {
 
 int HardwareBreakpointManager::hit(Target::Thread *thread, Site &site) {
   return -1;
-};
```

Comment:
The changes in this file are good. I'm not sure wtf these semicolons are doing there...